library/proc_macro/src/quote.rs

```diff
@@ -35,7 +37,7 @@ macro_rules! quote_ts {
             })
             .collect::<TokenStream>()
```

Comment:
While you're working on this file, mind cleaning up the code here? The combinator use is kind of unusual.

```rust
let mut c1 = TokenTree::from(Punct::new(':', Spacing::Joint));
let mut c2 TokenTree::from(Punct::new(':', Spacing::Alone));
c1.set_span(Span::def_site());
c2.set_span(Span::def_site());
[c1, c2].into_iter().collect::<TokenStream>()
```